src/h5web/test-utils.tsx

```diff
@@ -76,31 +76,26 @@ export function prepareForConsoleError() {
 }
 /* eslint-enable no-console */
 
-type Opts = Partial<Pick<MyHDF5Dataset, 'id' | 'parents' | 'attributes'>>;
+type Opts = Partial<Pick<MyHDF5Dataset, 'id' | 'attributes'>>;
```

Comment:
While I'm at it, I'm removing the ability to create a parent before its child (see in tests below).